packages/vsphere-csi/charts/Chart.yaml

```diff
@@ -9,3 +9,11 @@ icon: https://charts.rancher.io/assets/logos/vsphere-csi.svg
 maintainers:
   - name: Rancher
     email: caleb@rancher.com
+keywords:
+- infrastructure
+annotations:
+   catalog.cattle.io/certified: rancher
+   catalog.cattle.io/namespace: kube-system
+   catalog.cattle.io/release-name: vsphere-csi
+   catalog.cattle.io/display-name: "vSphere Cloud Storage Interface"
```

Comment:
@deniseschannon does this Display Name work or should it be short (e.g. `vSphere CSI`)

Reply:
vSphere Cloud Storage Interface (CSI)

Reply:
Updated, will update CPI as well.